scripts/cmux-chat-debug.py

```
@@ -0,0 +1,138 @@
+"""
+import json
+import os
+import re
+import subprocess
+import sys
+import time
+
+CLI = os.path.join(os.path.dirname(__file__), "cmux-debug-cli.sh")
+
+
+def cli(*args):
```

Ruff (0.15.18)

[warning] Missing type annotation for `*args`

(ANN002)